ssuslipstream/ssuslipstream.cpp

```diff
@@ -0,0 +1,66 @@
+/**
+ * @file ssuslipstream.cpp
+ * @copyright 2015 Jolla Ltd.
+ * @author Thomas Perl <thomas.perl@jolla.com>
+ * @date 2015
+ */
+
+#include "ssuslipstream.h"
+
+#include <QCoreApplication>
+#include <QTimer>
+#include <QUrl>
+
+#include "libssu/ssu.h"
+#include "libssu/ssudeviceinfo.h"
+#include "libssu/ssucoreconfig.h"
+
+
+SsuSlipstream::SsuSlipstream()
+    : QObject()
+{
+}
+
+void SsuSlipstream::run()
+{
+    Ssu ssu;
```

Comment:
This could be initialized later as the function can return without ever touching it.